### DISCUSSION TITLE
Eslint + Prettier

### DISCUSSION THREAD

**Discussion by acollazog11** [switchfeat-com/switchfeat#59] [General]
Hi @dev-bre.
I think it is better to use this space to discuss about the style guide rules.

let me know what do you think about this rules:
- semicolons (true)
- tab width (2 spaces)
- single quote
- print with (80)
- arrow parens (true)

**Comment by dev-bre** [comment]
Thanks for creating the discussion.

I would like to go with:

semicolons (true)
Use tab indentation 
tab width (4 spaces)
Double quotes for strings
print width (80)
arrow parens (true)

Run prettier on file save and fix the formatting automatically .

**Reply by acollazog11** [reply]
Perfect!!, I going to create an issue for that

**Reply by dev-bre** [reply]
#60

**Reply by acollazog11** [reply]
working on it!

**Comment by dev-bre** [comment]
Great! I already created an issue. Please use this one:

https://github.com/switchfeat-com/switchfeat/issues/60
